### DISCUSSION TITLE
Data importer connections for American institutions?

### DISCUSSION THREAD

**Discussion by nerlins** [firefly-iii/firefly-iii#7401] [Firefly III]
I don't know if I'm missing something, but neither Nordigan or Saltedge display the ability to connect to American institutions. I.e., Bank of America, Citibank, and so forth. I would love to use Firefly, but seems not useful if those connectors can't connect to American banks/institutions. So, am I missing something?

**Comment by JC5** [comment]
Salt Edge used to support American banks, but they stopped with all of them unfortunately.

**Comment by nerlins** [comment]
> Salt Edge used to support American banks, but they stopped with all of them unfortunately. 

Well, that stinks. So right now, the only way for a user with American sources to use Firefly is manual imports of all transaction data, correct?

**Reply by JC5** [reply]
As far as I know, yes.

**Reply by marleyjaffe** [reply]
☹️
